skbio/diversity/alpha/_base.py

```diff
@@ -311,6 +293,52 @@ def esty_ci(counts):
     return f1 / n - z * np.sqrt(W), f1 / n + z * np.sqrt(W)
 
 
+@experimental(as_of="0.4.0-dev")
+def faith_pd(counts, otu_ids, tree):
```

Comment:
Should this also have a `suppress_validation` like the `unweighted_unifrac` and `weighted_unifrac`?